README.md

```diff
@@ -15,75 +15,72 @@
 </p>
 <p>&nbsp;</p>
 
-Hops is a modular development, build and runtime environment for universal ("isomorphic") web applications. It focuses on [React](https://facebook.github.io/react/).
+Hops is everything you need to develop and deploy a production grade universal web application with [React](https://facebook.github.io/react/). It provides both a **universal runtime** as well as the necessary **build tooling**.
 
-Hops is designed to simplify getting started with modern frontend tooling. It streamlines [Webpack](https://webpack.js.org) and [Jest](https://facebook.github.io/jest/) configuration featuring [Babel](https://babeljs.io) and [PostCSS](http://postcss.org).
+### Philosophy
 
-### Quick start
+Hops targets the the professional React developer and follows React mainstream best practices
 
-If you just want to get started with hops + react you can execute the following command to initialize a small example app:
+The two guiding principles are
 
-If you have npm >= 5.2.0 installed:
+1. Hops scales with your requirements, from an easy start up to large scale applications with many teams
+1. Hops fits a broad need by being modular and extensible, it comes with reasonable defaults, but allows you to customize, configure, and extend almost everything when needed
 
-```shell
-npx hops init my-awesome-project
-cd my-awesome-project
-npm start
-```
+Hops is designed to simplify getting started with modern frontend tooling. It streamlines [Webpack](https://webpack.js.org) and [Jest](https://facebook.github.io/jest/) configuration featuring [Babel](https://babeljs.io) and [PostCSS](http://postcss.org).
```

Comment:
I'd also like to have this sentence revised, as Hops is about more than just Webpack and Jest and it also provides tooling for more than just Babel and PostCSS (and in the future potentially also TypeScript).

Reply:
I have simply taken this from the previous version. Should we just get rid of it?

Reply:
Yes, let's remove it for now and add something else later on.
I think it would be important to mention the core tools and frameworks that Hops is built upon / brings by default (like hops-react brings React and React-Router, and the build tooling supports Babel in stage X / env plugin and these other plugins... PostCSS with CSS Modules and CSS Next support, etc) - maybe not that much in depth but still explain to the user what they can do with hops? Similar to: https://github.com/facebook/create-react-app#whats-included

Reply:
+1